exo-setup/src/docker-setup.ls

```diff
@@ -26,8 +25,7 @@ class DockerSetup extends EventEmitter
 
 
   _build-docker-image: (done) ~>
-    service-name = @config.root.split path.sep |> last
-    new ObservableProcess(call-args(DockerHelper.get-build-command author: @service-config.author, name: service-name),
+    new ObservableProcess(call-args(DockerHelper.get-build-command author: @service-config.author, name: dashify @service-config.title),
```

Comment:
I would put parentheses for the dashify call, so that it is more clear that we have a method call here.

```ls
new ObservableProcess call-args(DockerHelper.get-build-command author: @service-config.author, name: dashify(@service-config.title),